example/simple/test_comparisions.tavern.yaml

```diff
@@ -0,0 +1,19 @@
+---
+
+test_name: test_comparisions
+
+stages:
+  - name: test_comparisions
+    request:
+      url: https://jsonplaceholder.typicode.com/posts/1
+      method: GET
+    response:
+      status_code: 200
+      body:
+        id: 1
+        $ext:
+          function: helper:validate_content
+          extra_kwargs:
+            comparisions:
+              - {jmespath: "userId", operator: "eq", expected: 1}
+              - {jmespath: "userId", operator: "type", expected: str}
```

Comment:
Could you add a few more integration tests against the local test server as well? The `/fake_dictionary` endpoint returns a dictionary like this:

```json
{
        "top": {
            "Thing": "value",
            "nested": {
                "doubly": {
                    "inner": "value",
                }
            }
        },
        "an_integer": 123,
        "a_string": "abc",
        "a_bool": true
    }
```
so you could check that accessing nested keys works as well.